configserver/src/main/java/com/yahoo/vespa/config/server/rpc/RpcServer.java

```diff
@@ -102,6 +103,14 @@
     private volatile boolean allTenantsLoaded = false;
     private boolean isRunning = false;
 
+    class ApplicationState {
```

Comment:
This doesn't look like it needs to be an inner class, so it should be declared static.

Reply:
Thanks, fixed in #8038